lib/pangea/controllers/permissions_controller.dart

```diff
@@ -36,63 +35,73 @@ class PermissionsController extends BaseController {
     return dob?.isAtLeastYearsOld(AgeLimits.toAccessFeatures) ?? false;
   }
 
-  /// A user can private chat if
-  /// 1) they are 18 and outside a class context or
-  /// 2) they are in a class context and the class rules permit it
-  /// If no class is passed, uses classController.activeClass
+  /// A user can private chat if they are 18+
   bool canUserPrivateChat({String? roomID}) {
-    final Room? classContext =
-        firstRoomWithState(roomID: roomID, type: PangeaEventTypes.rules);
-    return classContext?.pangeaRoomRules == null
-        ? isUser18()
-        : classContext!.pangeaRoomRules!.oneToOneChatClass ||
-            classContext.isRoomAdmin;
+    return isUser18();
+    // Rules can't be edited; default to true
+    // final Room? classContext =
+    //     firstRoomWithState(roomID: roomID, type: PangeaEventTypes.rules);
+    // return classContext?.pangeaRoomRules == null
+    //     ? isUser18()
+    //     : classContext!.pangeaRoomRules!.oneToOneChatClass ||
+    //         classContext.isRoomAdmin;
   }
 
   bool canUserGroupChat({String? roomID}) {
-    final Room? classContext =
-        firstRoomWithState(roomID: roomID, type: PangeaEventTypes.rules);
-
-    return classContext?.pangeaRoomRules == null
-        ? isUser18()
-        : classContext!.pangeaRoomRules!.isCreateRooms ||
-            classContext.isRoomAdmin;
+    return isUser18();
+    // Rules can't be edited; default to true
+    // final Room? classContext =
+    //     firstRoomWithState(roomID: roomID, type: PangeaEventTypes.rules);
+
+    // return classContext?.pangeaRoomRules == null
+    //     ? isUser18()
+    //     : classContext!.pangeaRoomRules!.isCreateRooms ||
+    //         classContext.isRoomAdmin;
   }
 
   bool showChatInputAddButton(String roomId) {
     final PangeaRoomRules? perms = _getRoomRules(roomId);
```

Comment:
Why is this one different from the rest?

Reply:
Are you referring to showChatInputAddButton, or the function above it?

Reply:
If you were referring to showChatInputAddButton, I just pushed a commit for that.

Reply:
Yep! Thank you. I will check on running this tomorrow morning.